ProductsWebApi/ProductsWebApi.Tests/ProductsServiceTest.cs

```diff
@@ -0,0 +1,123 @@
+﻿using AutoFixture;
+using FluentAssertions;
+using Moq;
+using ProductsWebApi.Interface;
+using ProductsWebApi.Models;
+using ProductsWebApi.Services;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace ProductWebApi.Tests
+{
+    public class ProductsServiceTest
+    {
+        private readonly Mock<IProductsService> _mockproductService;
+        private readonly ProductsService _productService;
+        private readonly Fixture _fixture;
+        private string expectedNullResult = null;
+        private Product productRequestObj = null;
```

Comment:
Always prefer full naming. 
productRequestObject.
In this case, 
it can be productRequest